examples/guided.py

```diff
@@ -365,31 +494,44 @@ def perform_installation(mountpoint):
 			for superuser, user_info in archinstall.arguments.get('superusers', {}).items():
 				installation.user_create(superuser, user_info["!password"], sudo=True)
 
-			if (timezone := archinstall.arguments.get('timezone', None)):
+			if (timezone := archinstall.arguments.get('timezone', None)) :
 				installation.set_timezone(timezone)
 
 			if (root_pw := archinstall.arguments.get('!root-password', None)) and len(root_pw):
 				installation.user_set_pw('root', root_pw)
 
-			if archinstall.arguments['profile'] and archinstall.arguments['profile'].has_post_install():
-				with archinstall.arguments['profile'].load_instructions(namespace=f"{archinstall.arguments['profile'].namespace}.py") as imported:
+			if (
+				archinstall.arguments['profile']
+				and archinstall.arguments['profile'].has_post_install()
+			):
+				with archinstall.arguments['profile'].load_instructions(
+					namespace=f"{archinstall.arguments['profile'].namespace}.py"
+				) as imported:
 					if not imported._post_install():
 						archinstall.log(
-							' * Profile\'s post configuration requirements was not fulfilled.',
-							fg='red'
+							' * Profile\'s post configuration requirements was not fulfilled.', fg='red'
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Again, linter doing the complete opposite of what it's been doing before..